services/venice-standalone/src/main/java/com/linkedin/venice/standalone/VeniceStandalone.java

```diff
@@ -35,4 +68,72 @@ public static void main(String[] args) throws Exception {
     }
 
   }
+
+  private static boolean isServiceEnabled(String service) {
+    String services = System.getProperty("services", "*");
+    if (services.equals("*") || services.toLowerCase().contains(service)) {
+      return true;
+    }
+    return false;
+  }
+
+  private static void startZooKeeperServer(String[] args) throws Exception {
+    File zookeeperConfig = new File(args[0], "zookeeper.properties");
+    if (zookeeperConfig.isFile()) {
+      Properties properties = new Properties();
+      try (Reader reader = new FileReader(zookeeperConfig)) {
+        properties.load(reader);
+      }
+      boolean enabled = Boolean.parseBoolean(properties.getProperty("enabled", "false"));
+      if (enabled) {
+        String dataDirectory = properties.getProperty("dataDir", "zookeeperData");
+        int port = Integer.parseInt(properties.getProperty("port", "2181"));
+        LOGGER.info("Starting ZooKeeper on port {}, dataDir is {}", port, dataDirectory);
+        InstanceSpec spec = new InstanceSpec(new File(dataDirectory), port, -1, -1, false, 1);
+        TestingServer zkServer = new TestingServer(spec, false);
```

Comment:
I've heard curator is easy to use but I wonder whether we can just use the [ZKServerWrapper](https://github.com/linkedin/venice/blob/9653d18ca78d69f972f3dc5b2fa3a8dde0c11966/internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/integration/utils/ZkServerWrapper.java#L29) we have been using in the integ tests.

My concern is having `curator` would introducing duplicate dependencies on Zk and might increase some overhead/confusion in the future

Reply:
This is only for Standalone.

A better alternative would be to upgrade to ZooKeeper 3.7+

Reply:
Am open to considering the ZK upgrade. I would just like to get some input from our ZK team about whether upgrading the client to 3.7 ahead of the ZK server could cause any issues.

I'm curious though, why would upgrading to 3.7 help here? Is there better support for spinning it up standalone in that version?